csharp/CSharpParser.g4

```diff
@@ -691,7 +691,7 @@ common_member_declaration
 	| event_declaration
 	| conversion_operator_declarator (body | right_arrow throwable_expression ';') // C# 6
 	| constructor_declaration
-	| VOID method_declaration
+	| method_declaration  // 2020/10/22 add return_type
```

Comment:
Git keeps history, that's why such a comment is excess.

Reply:
done